src/Resources/Locales/de_DE.axaml

```diff
@@ -107,7 +107,7 @@
   <x:String x:Key="Text.CommitCM.Revert" xml:space="preserve">Commit rückgängig machen</x:String>
   <x:String x:Key="Text.CommitCM.Reword" xml:space="preserve">Umformulieren</x:String>
   <x:String x:Key="Text.CommitCM.SaveAsPatch" xml:space="preserve">Als Patch speichern...</x:String>
-  <x:String x:Key="Text.CommitCM.Squash" xml:space="preserve">Squash in den Parent</x:String>
+  <x:String x:Key="Text.CommitCM.Squash" xml:space="preserve">Squash in den Vorgänger</x:String>
```

Comment:
"Vorgänger" could also mean "previous sibling". Maybe "Übergeordneten"?

Reply:
I thought about this for a while now. I am not sure if I fully understand what you mean with "previous sibling". Looking at the image below: If I am on commit c3 and then "Squash in den Vorgänger" it could be assumed that the commit would be squashed into c2 instead of c1? 

![grafik](https://github.com/user-attachments/assets/2fef0a9a-f9a6-4635-af90-f49c274cb476)

The things is "Übergeordneten" is not a bad translation but it's just such a long word 😅

Reply:
Yes (to me), although with this image it gets rather clear that C2 would have nothing to do about it.
So then I also thought about it a little bit more and can see that this may be clear enough from the context. So "Vorgänger" is fine. 👍